### PR TITLE
Feature/dashboard

### PR DESCRIPTION
**Pull request by alexsicart**

### CODE REVIEW

File: package.json
Comment (jmporchet) [inline package.json]: do we sill need faker?

File: package.json
Comment (jmporchet) [inline package.json]: faker?

File: package.json
Comment (jmporchet) [inline package.json]: Where is this package used?

File: public/style.css
Comment (jmporchet) [inline public/style.css]: Is this file really needed?

File: src/components/Dashboard/card.js
Comment (jmporchet) [inline src/components/Dashboard/card.js]: Use a more descriptive class name like InstagramCard or LikeCard

File: src/components/Dashboard/card.js
Comment (jmporchet) [inline src/components/Dashboard/card.js]: Don't hardcode the token

``` 'Authorization': 'Bearer ' + this.props.access_token```

File: src/components/Dashboard/card.js
Comment (jmporchet) [inline src/components/Dashboard/card.js]: remove comment

File: src/components/Dashboard/card.js
Comment (jmporchet) [inline src/components/Dashboard/card.js]: remove comment

File: src/components/Dashboard/card.js
Comment (jmporchet) [inline src/components/Dashboard/card.js]: change this

File: src/components/Dashboard/card.js
Comment (jmporchet) [inline src/components/Dashboard/card.js]: token

File: src/private/conf.js
Comment (jmporchet) [inline src/private/conf.js]: This file should have been .gitignored
Don't expose these secrets on Github

File: src/components/Dashboard/fakeData.js
Comment (jmporchet) [inline src/components/Dashboard/fakeData.js]: Do we still need this file?

File: package.json
Comment (alexsicart) [reply package.json]: Done

File: package.json
Comment (alexsicart) [reply package.json]: Done

File: package.json
Comment (alexsicart) [reply package.json]: Object assign it is in use, json schema not

File: public/style.css
Comment (alexsicart) [reply public/style.css]: Moved to trash!

File: src/components/Dashboard/card.js
Comment (alexsicart) [reply src/components/Dashboard/card.js]: Changed to InstagramCard

File: src/components/Dashboard/card.js
Comment (alexsicart) [reply src/components/Dashboard/card.js]: done

File: src/components/Dashboard/card.js
Comment (alexsicart) [reply src/components/Dashboard/card.js]: done

File: src/components/Dashboard/card.js
Comment (alexsicart) [reply src/components/Dashboard/card.js]: done

File: src/components/Dashboard/card.js
Comment (alexsicart) [reply src/components/Dashboard/card.js]: Changed

File: src/components/Dashboard/card.js
Comment (alexsicart) [reply src/components/Dashboard/card.js]: Changed

File: src/components/Dashboard/fakeData.js
Comment (alexsicart) [reply src/components/Dashboard/fakeData.js]: Deleted

**Comment by alexsicart** [comment]
The app at this moment is working correctly, without any conflict.
Changes made:
- Installing the correct version of "react-key-handler", it didn't let the app work correctly.
- Deleting package-lock and merged package-lock.json from develop
- Minor big fixes that made the app more stressful.

Waiting for a review from @arol & @jmporchet 

Looking forward this changes be good.


AS

File: package.json
Comment (arol) [inline package.json]: Come back to 3001, we want to run this at the same time as the back-end

**Review by arol — DISMISSED** [review]

File: package.json
Comment (alexsicart) [reply package.json]: Completed @arol Thanks!

**Comment by alexsicart** [comment]
All conflicts solve, and requested changes completed.

**Comment by alexsicart** [comment]
The last request for @arol was done, and the merge branch 'develop' into this branch was done.

AS

**Review by jmporchet — APPROVED** [review]

**Pull request merged**
